### PR TITLE
1.10-7.4

### PR DESCRIPTION
**Pull request by Stolidbug**
update 1.10-7.4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ return [
 ```yaml
 # config/packages/sylius_cookie_alert.yaml
 imports:
-    - { resource: "@BlackSyliusCookieAlertPlugin/Resources/config/app/config.yml" }
+    - { resource: "@BlackSyliusCookieAlertPlugin/Resources/config/app/config.php" }
```
Comment (pocky) [inline README.md]: Invalid path

File: composer.json [hunk 2]
```diff
@@ -12,33 +12,38 @@
         }
     ],
     "require": {
-        "php": "^7.3",
-        "sylius/sylius": "^1.7"
+        "php": "7.4.21",
+        "sylius/sylius": "1.10.0"
```
Comment (pocky) [inline composer.json]: should be ~1.10.0

File: composer.json [hunk 3]
```diff
@@ -12,33 +12,38 @@
         }
     ],
     "require": {
-        "php": "^7.3",
-        "sylius/sylius": "^1.7"
+        "php": "7.4.21",
```
Comment (pocky) [inline composer.json]: Should be ^7.4

File: config/config.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return function (ContainerConfigurator $configurator) {
+
+    $configurator->events(sylius.shop.layout.after_body)
+        ->blocks(cookie)
+        ->template("@BlackSyliusCookieAlertPlugin/_cookieAlert.html.twig");
+};
+
+
```
Comment (pocky) [inline config/config.php]: Remove this line

File: config/config.php [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+    declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return function (ContainerConfigurator $configurator) {
+
+    $configurator->events(sylius.shop.layout.after_body)
+        ->blocks(cookie)
```
Comment (pocky) [inline config/config.php]: missing quotes on theses lines

File: config/routes/admin_routing.yaml [hunk 6]
```diff
@@ -0,0 +1 @@
+# Define your own admin routes here
```
Comment (pocky) [inline config/routes/admin_routing.yaml]: remove this file

File: config/routes/shop_routing.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+    path: /static-welcome/{name}
+    defaults:
+        _controller: Acme\SyliusExamplePlugin\Controller\GreetingController::staticallyGreetAction
+        name: ~
+
+acme_sylius_example_dynamic_welcome:
+    path: /dynamic-welcome/{name}
+    defaults:
+        _controller: Acme\SyliusExamplePlugin\Controller\GreetingController::dynamicallyGreetAction
+        name: ~
```
Comment (pocky) [inline config/routes/shop_routing.yaml]: remove this file

File: src/DependencyInjection/BlackSyliusCookieAlertExtension.php [hunk 8]
```diff
@@ -16,7 +16,7 @@ final class BlackSyliusCookieAlertExtension extends AbstractResourceExtension
     public function load(array $config, ContainerBuilder $container): void
     {
         $config = $this->processConfiguration($this->getConfiguration([], $container), $config);
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '../../config'));
```
Comment (pocky) [inline src/DependencyInjection/BlackSyliusCookieAlertExtension.php]: Cant be XmlFileLoader, your configuration is in php

File: config/routes/admin_routing.yaml (hunk 6 above)
Comment (Stolidbug) [reply config/routes/admin_routing.yaml]: check

File: composer.json (hunk 3 above)
Comment (Stolidbug) [reply composer.json]: check

File: composer.json (hunk 2 above)
Comment (Stolidbug) [reply composer.json]: check

File: README.md (hunk 1 above)
Comment (Stolidbug) [reply README.md]: check

File: config/config.php (hunk 4 above)
Comment (Stolidbug) [reply config/config.php]: check

File: config/config.php (hunk 5 above)
Comment (Stolidbug) [reply config/config.php]: check

File: config/routes/shop_routing.yaml (hunk 7 above)
Comment (Stolidbug) [reply config/routes/shop_routing.yaml]: check

File: src/DependencyInjection/BlackSyliusCookieAlertExtension.php (hunk 8 above)
Comment (Stolidbug) [reply src/DependencyInjection/BlackSyliusCookieAlertExtension.php]: check

File: config/app/config.php [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    'events' => [
+        'sylius.shop.layout.after_body'     => [
+            'blocks' => ['cookie' => [
+                'template' => '@BlackSyliusCookieAlertPlugin/_cookieAlert.html.twig'
+
+            ]],
+        ]
+    ],
+])
+;
```
Comment (pocky) [inline config/app/config.php]: ";" should be on previous line

File: config/services.php [hunk 10]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return static function (ContainerConfigurator $configurator) {
+    $configurator->import('services/*');
+};
```
Comment (pocky) [inline config/services.php:9]: missing EOL

File: src/BlackSyliusCookieAlertPlugin.php [hunk 11]
```diff
@@ -10,4 +10,9 @@
 final class BlackSyliusCookieAlertPlugin extends Bundle
 {
     use SyliusPluginTrait;
+
+    public function getPath(): string
+    {
+        return \dirname(__DIR__);
+    }
 }
```
Comment (pocky) [inline src/BlackSyliusCookieAlertPlugin.php:18]: Missing EOL

File: src/DependencyInjection/BlackSyliusCookieAlertExtension.php [hunk 12]
```diff
@@ -16,9 +16,9 @@ final class BlackSyliusCookieAlertExtension extends AbstractResourceExtension
...
     {
         $config = $this->processConfiguration($this->getConfiguration([], $container), $config);
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+        $loader = new PhpFileLoader($container, new FileLocator(__DIR__ . '/../../config'));
         $this->registerResources('black_sylius_cookie_alert', $config['driver'], $config['resources'], $container);
 
-        $loader->load('services.xml');
+        $loader->load('services.php');
     }
 }
```
Comment (pocky) [inline src/DependencyInjection/BlackSyliusCookieAlertExtension.php:24]: Missing EOL

File: README.md [hunk 13]
```diff
@@ -37,7 +37,7 @@ return [
 ```yaml
 # config/packages/sylius_cookie_alert.yaml
 imports:
-    - { resource: "@BlackSyliusCookieAlertPlugin/Resources/config/app/config.yml" }
+    - { resource: "@BlackSyliusCookieAlertPlugin/config/config.php" }
```
Comment (pocky) [inline README.md:40]: Wrong path

File: README.md [hunk 14]
```diff
@@ -69,7 +69,6 @@ sylius_ui:
 
 ## License and Copyright
 
-tl;dr:
```
Comment (pocky) [inline README.md:72]: porque?

File: config/app/config.php (hunk 9 above)
Comment (Stolidbug) [reply config/app/config.php]: ok

File: config/services.php (hunk 10 above)
Comment (Stolidbug) [reply config/services.php:9]: check

File: src/BlackSyliusCookieAlertPlugin.php (hunk 11 above)
Comment (Stolidbug) [reply src/BlackSyliusCookieAlertPlugin.php:18]: check

File: src/DependencyInjection/BlackSyliusCookieAlertExtension.php (hunk 12 above)
Comment (Stolidbug) [reply src/DependencyInjection/BlackSyliusCookieAlertExtension.php:24]: check

File: README.md (hunk 13 above)
Comment (Stolidbug) [reply README.md:40]: check

File: README.md (hunk 14 above)
Comment (Stolidbug) [reply README.md:72]: check

**Pull request merged**